### ISSUE TITLE
appveyor.yml: GUID's password for connecting to Azure should be removed

### ISSUE THREAD

**Issue by johlju** [PowerShell/TemplateConfig#8]
We should not encourage to save GUID's and passwords in the appveyor.yml. Even if they are stored encrypted, they can be copied to another repository and run against AppVeyor to be decrypted.

Instead we should add documentation for how to add these GUID's and password to the configuration in the AppVeyor portal.

![image](https://user-images.githubusercontent.com/7189721/28972328-9563f5a2-792f-11e7-8128-dbf62b9290e3.png)

**Comment by PlagueHO** [comment]
Agreed! This isn't a good security practice - never store any kind of credential information (encrypted or not) in a repo. So this should be changed ASAP.

That said, I think they can only be decrypted in the AppVeyor account they were created in - e.g. if I create them in my AppVeyor account, only my account can decrypt them - but I think they must be marked as encrypted.

**Comment by johlju** [comment]
Yes, seems you are correct regarding being encrypt per account. If I go to https://ci.appveyor.com/tools/encrypt I have to log on with an account, which seem to suggest that it is my account that is encrypting them and another account can't decrypt them. 

But as you also say, it still would be good not to have these in the AppVeyor.yml file.
